modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc

```diff
@@ -57,82 +57,142 @@ podman login -u kubeadmin -p $(oc whoami -t) $REGISTRY_HOST
 podman login registry.redhat.io
 ----
 +
-For more information about registry authentication, see https://access.redhat.com/RegistryAuthentication[{company-name} Container Registry Authentication].
-
-. Pull {product-short} and PostgreSQL images from https://catalog.redhat.com/software/containers/search[{company-name} Image registry] to your workstation, by running the following commands:
+. Create an `ImageSetConfiguration` file to specify the resources that you want to mirror. For example:
++
+[source,terminal]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+storageConfig:
+  local:
+    path: ./mirror-output
+mirror:
+  helm:
+    repositories:
+      - name: openshift-charts
+        url: https://charts.openshift.io
+        charts:
+          - name: redhat-developer-hub
+            version: "{product-version}"
+----
+. Mirror the resources specified in the`ImageSetConfiguration` file directly to the target registry by running the following command:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal]
 ----
-podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-version}
+oc-mirror --config=mirror-config.yaml file://mirror-archive
 ----
+. In your air-gapped environment, deploy the Helm chart to the `rhdh` namespace by running the `helm install` command. For example:
 +
-[source,terminal,source,subs="attributes+"]
+[source,terminal,subs="attributes+"]
 ----
-podman pull registry.redhat.io/rhel9/postgresql-15:latest
+helm install rhdh ./mirror-archive/oc-mirror-workspace/src/charts/redhat-developer-hub-{product-version}.tgz --namespace rhdh --create-namespace
 ----
 
-. Push both images to the internal {ocp-short} image registry by running the following commands:
+.Verification
+. To verify that all resources are successfully deployed and running, check the status of your deployments and pods in the cluster.
+. To validate the route, ensure that the route matches the expected URL based on the router base of the cluster.
+.. If necessary, update the route to align with your expected configuration.
```

Comment:
How do we expect users to validate and update this?
How about using a similar procedure as what we have for "[Deploying Developer Hub on OCP with the Helm CLI](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.4/html/installing_red_hat_developer_hub_on_openshift_container_platform/assembly-install-rhdh-ocp-helm#proc-install-rhdh-ocp-helm-cli_assembly-install-rhdh-ocp-helm)", by providing the command for the user to get the right router base URL? Like so:

```sh
CLUSTER_ROUTER_BASE=$(oc get route console -n openshift-console -o=jsonpath='{.spec.host}' | sed 's/^[^.]*\.//')
helm install redhat-developer-hub./mirror-archive/oc-mirror-workspace/src/charts/redhat-developer-hub-1.4.1.tgz \
    --set global.clusterRouterBase="$CLUSTER_ROUTER_BASE"
```